test/test.js

```diff
@@ -111,6 +111,33 @@ async function run () {
     t.deepEqual(resultRoot.body, { something: 'posted' })
   })
 
+  test('skip content parsing', async t => {
+    let requestBody
+    const server = Fastify()
+    server.register(proxy, {
+      upstream: `http://localhost:${origin.server.address().port}`,
+      skipContentParsing: false,
```

Comment:
```suggestion
      proxyPayloads: false,
```

The `skipContentParsing: false` implies that content parsing _will not_ be skipped. `proxyPayloads: false` actually represents what is happening.

Reply:
Hi,
Even if the payload is not parsed it would be proxied to the upstream. So, here it would be more about parsing the payload so that you have access to the payload in something like a preHandler.

For the following usecase, where requests are coming into a gateway kind of service which proxys the call to the underlying services and wants to deny certain requests,
preHandler(req, reply, next) => {
    if(req.body.title === 'developer'){
        //deny that particular action
    }
    next();
}